lib/tasks/mastodon.rake

```diff
@@ -519,14 +537,46 @@ namespace :mastodon do
 
           Setting.site_contact_username = username
 
+          prompt.say "\n"
           prompt.ok "You can login with the password: #{password}"
           prompt.warn 'You can change your password once you login.'
+        else
+          prompt.warn 'Okay.'
+        end
+
+        prompt.say "\n"
+
+        blocklist = prompt.select("Would you like to import a domain blocklist for your instance?\n") do |menu|
```

Comment:
We could change this to a 2-3 step process:

```
Would you like to import a domain blocklist for your instance? Yn

Which domain blocklist would you like to import?
1. Oliphant (website)
2. The Bad Space (website)
3. Other

# If `Other` then:
What is the URL of blocklist you would like to import?
What is the name of the blocklist?
```

As to allow importing something stricter than the basic ones proposed above in [`domain_blocklists`](https://github.com/mastodon/mastodon/pull/25691/files#diff-0be915342a07d91f53ec7039d97ef9138f5e96e82c01d95a4de35cfce413d608R12)

Reply:
I guess I should also add an explainer like

> "As Mastodon is connected to a network of other servers and services, there's a small minority of servers that are widely known to have problematic users or behaviour, over time, other administrators & moderators have collated lists of these servers such that new administrators can easily block the worst of the worst before your server even joins the fediverse, keeping you and people using your server safe"